Makefile

```diff
@@ -10,25 +10,28 @@ else
 DOCKER_IMAGE_NAME := "docker-machine-build"
 DOCKER_CONTAINER_NAME := "docker-machine-build-container"
 
+.ignore:
+	@
+
 build:
-test: build
+test: .ignore
```

Comment:
Need a phony target, otherwise make will bail on it since we have a test folder that exists.